src/applications/personalization/dashboard/tests/e2e/benefits-of-interest.cypress.spec.js

```diff
@@ -108,6 +109,55 @@ describe('The My VA Dashboard', () => {
       disabilityCompensationExists(true);
       educationBenefitExists(false);
 
+      cy.injectAxe();
+      cy.axeCheck();
+    });
+  });
+  describe('when user is not a patient, has not applied for health care, and applied to education benefits', () => {
+    beforeEach(() => {
+      const user = makeMockUser();
+      const now = Date.now() / 1000;
+      const oneDayInSeconds = 24 * 60 * 60;
+      const savedForms = [
+        // This form is unknown and will be filtered out of the list of applications
+        {
+          form: VA_FORM_IDS.FORM_22_1990E,
+          metadata: {
+            version: 0,
+            returnUrl: '/education-benefits',
+            savedAt: 1611946775267,
+            submission: {
+              status: false,
+              errorMessage: false,
+              id: false,
+              timestamp: false,
+              hasAttemptedSubmit: false,
+            },
+            expiresAt: now + oneDayInSeconds,
+            lastUpdated: 1611946775,
+            inProgressFormId: 9332,
+          },
+          lastUpdated: 1611946775,
+        },
+      ];
+      user.data.attributes.vaProfile.vaPatient = false;
+      user.data.attributes.vaProfile.facilities = [];
+      user.data.attributes.inProgressForms = savedForms;
+      cy.intercept('/v0/health_care_applications/enrollment_status', notInESR);
+      cy.intercept('/v0/profile/ch33_bank_accounts', dd4eduNotEnrolled);
```

Comment:
nit: This mock isn't actually required since this api won't be called when a user has an EDU form in progress. And if you wanted to, you could add a check to _make sure_ that the API isn't called in this case. Something like this snippet from another test:
```
// set up an empty var to hold a stub
let getPaymentInfoStub
beforeEach(() => {
  // set the stub before each test
  getPaymentInfoStub = cy.stub()
})
it('should be blocked if the user is not in EVSS and they are not signed up for DD4EDU', () => {
  // intercept the endpoint you want to spy on to make sure it's not called
  cy.intercept('v0/ppiu/payment_information', () => {
    getPaymentInfoStub()
  })
  // make sure the stub is not called (wrapped in a cy.should callback)
  cy.should(() => {
    expect(getPaymentInfoStub).not.to.be.called
  })
})
```